src/plugins/home/public/application/components/_add_data.scss

```diff
@@ -17,6 +17,17 @@
  * under the License.
  */
 
+.homDataAdd__actions {
+  // Prevent children from rendering as flex items
+  display: block;
```

Comment:
All you have to do to accomplish this in JS is, inside the EuiFlexItem, add div wrapper to the children.

```tsx
<EuiFlexItem><div>{children}</div></EuiFlexItem>
```

Reply:
Hey, @cchaos! This one is on me again. In situations such as this, where it's not desirable for an `EuiFlexItem` to be `display: flex`, my first instinct was to just override the style directly, rather than add additional markup to the page in order to achieve the style I'm looking for. However, if you strongly disagree with this thinking, I'll defer to your judgement. Thoughts?

Reply:
Either way does work, I just find that using the wrapping div tends to be more scalable and maintainable than overriding the styles as well as more obvious to others looking at the file.

Reply:
Sounds good. I'll see about making this change.